advtraj/utils/interpolation.py

```diff
@@ -62,7 +62,7 @@ def interpolate_3d_field(da, ds_positions, interp_order=1, cyclic_boundaries=[])
         a=c_min, b=c_max, h=dX, f=da.values, k=interp_order, p=periodicity
     )
 
-    vals = fn(*[ds_positions[c] for c in da.dims])
+    vals = fn(*[ds_positions[c].values for c in da.dims])
```

Comment:
I wonder why this didn't with what was there before?

Reply:
I think it worked ok with only a single trajectory. Incidentally, this is a very neat bit of code but, not surprisoingly, shows up as a bottlneck when profiling. At some point it may be worth seeing if there's a faster solution.